tests/integration/queries/tpch/query_1.py

```diff
@@ -0,0 +1,844 @@
+import pytest
+
+TESTCASE = [
```

Comment:
I originally put these as python files so that I could use some pytest decorators for marking tests, i.e., skipping one that fails due to a known bug.

I can separate the substrait plan portion into it's own json file if that makes things easier.